specificatie/beslagen.yaml

```diff
@@ -157,12 +157,14 @@ components:
         _links:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/HalCollectionLinks"
         _embedded:
-          type: "object"
-          properties:
-            beslagen:
-              type: "array"
-              items:
-                $ref: "#/components/schemas/BeslagHal"
+          $ref: "#/components/schemas/BeslagHalCollectie_embedded"
```

Comment:
Bij BRP gebruiken we deze constructie niet meer. Als de dezelfde DD zou toepassen, dan zou het BeslagHalCollectieEmbedded moeten zijn

Reply:
Scherp, ik pas ze aan. (en dan ook overal.......)